docs/design/autoware-architecture/localization/index.md

```diff
@@ -1,5 +1,300 @@
+
+#### Situations that can make the system unstable
+
+- White lines are scratchy or covered by rain or snow
+- Tight curvature such as intersections
+- Large reflection change of the road surface caused by rain or paint
+
+#### Functionalities
+
+- Correct vehicle positions along the lateral direction.
+- Pose correction along the longitudinal can be inaccurate, but can be resolved by fusing with GNSS.
+
+### GNSS
+
+#### Expected situation
+
+- The vehicle is placed in an open environment with few to no surrounding objects, such as a rural landscape.
+
+#### Situation that can make the system unstable
+
+- GNSS signals are blocked by surrounding objects, e.g., tunnels or buildings.
+
+#### Functionality
+
+- The system can estimate vehicle position in the world coordinate within an error of ~10m.
+- With RTK attached, the accuracy can be improved to ~10cm
+- The system with this configuration can work without environment maps
+
+### Camera (Visual Odometry, Visual SLAM)
+
+#### Expected situation
+
+- The vehicle is placed in a texture-rich environment, such as urban areas
+
+#### Situations that can make the system unstable
+
+- The vehicle is placed in a Texture-less environment
+- The vehicle is surrounded by other objects.
+- The camera observes significant illumination changes, such as those caused by sunshine, headlights from other vehicles or when approaching the exit of a tunnel.
+- The vehicle is placed in a dark environment.
+
+#### Functionality
+
+- The system can estimate odometry by tracking visual features.
+
+### Wheel speed sensor
+
+#### Expected situation
+
+- The vehicle is running on an ordinary road
+
+#### Situations that can make the system unstable
+
+- The vehicle is running on a slippery or bumpy road, which can cause incorrect observations of wheel speed.
+
+#### Functionality
+
+- The system can acquire the vehicle velocity and estimate distance traveled.
+- The system can estimate more accurate poses at a higher frequency by fusing with external sensors.
+
+<!-- NOTE: Is the second item necessary? Because it is general for any other sensors -->
+
+### IMU
+
+#### Expected environments
+
+- Ordinary roads
+
+<!-- NOTE: The availability of IMU is highest among the sensors attached to vehicles. Therefore we should list more situations or say we can use the IMU almost anywhere -->
+
+#### Situations that can make the system unstable
+
+- IMUs have a bias that is dependent on the surrounding temperature. This can cause incorrect sensor observation or odometry drift
+
+#### Functionality
+
+- The system can observe acceleration and angular velocity.
+- By integrating these observations, the system can estimate the local pose change and realize dead-reckoning
+- By fusing with external sensors, it can estimate more accurate poses in a higher frequency
+
+<!-- NOTE: The third item says "it can estimate more accurate poses in a higher frequency," but the IMU already has very frequent observation -->
```

Comment:
Contents of the note need to be actioned and then the note removed.

Reply:
@YamatoAndo @e-takeuchi How about this? IMU already has a very high frequency, so I think it is not really necessary to describe the fusion with other sensors

Reply:
@IshitaTakeshi I see. OK